core/src/main/resources/reference.conf

```diff
@@ -0,0 +1,380 @@
+            # The following are strategies used to make those links.  They are ordered.  If one fails
+            # The next one is tried:
+            #
+            # hard-link: attempt to create a hard-link to the file
+            # copy: copy the file
+            # soft-link: create a symbolic link to the file
+            #
+            # NOTE: soft-link will be skipped for Docker jobs
+            localization: [
+              "hard-link", "soft-link", "copy"
+            ]
+          }
+        }
+      }
+    }
+
+    #SGE {
+    #  actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+    #  config {
+    #    runtime-attributes = """
+    #    Int cpu = 1
+    #    Float? memory_gb
+    #    String? sge_queue
+    #    String? sge_project
+    #    """
+    #
+    #    submit = """
+    #    qsub \
+    #    -terse \
+    #    -V \
+    #    -b n \
+    #    -N ${job_name} \
+    #    -wd ${cwd} \
+    #    -o ${out} \
+    #    -e ${err} \
+    #    -pe smp ${cpu} \
+    #    ${"-l m_mem_free=" + memory_gb + "gb"} \
+    #    ${"-q " + sge_queue} \
+    #    ${"-P " + sge_project} \
+    #    ${script}
+    #    """
+    #
+    #    kill = "qdel ${job_id}"
+    #    check-alive = "qstat -j ${job_id}"
+    #    job-id-regex = "(\\d+)"
+    #  }
+    #}
+
+    #LSF {
+    #  actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+    #  config {
+    #    submit = "bsub -J ${job_name} -cwd ${cwd} -o ${out} -e ${err} /bin/bash ${script}"
+    #    kill = "bkill ${job_id}"
+    #    check-alive = "bjobs ${job_id}"
+    #    job-id-regex = "Job <(\\d+)>.*"
+    #  }
+    #}
+
+    # Example backend that _only_ runs workflows that specify docker for every command.
+    #Docker {
+    #  actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+    #  config {
+    #    run-in-background = true
+    #    runtime-attributes = "String docker"
+    #    submit-docker = "docker run --rm -v ${cwd}:${docker_cwd} -i ${docker} /bin/bash < ${script}"
+    #  }
+    #}
+
+    #HtCondor {
+    #  actor-factory = "cromwell.backend.impl.htcondor.HtCondorBackendFactory"
+    #  config {
+    #    # Root directory where Cromwell writes job results.  This directory must be
+    #    # visible and writeable by the Cromwell process as well as the jobs that Cromwell
+    #    # launches.
+    #    root: "cromwell-executions"
+    #
+    #    cache {
+    #      provider = "cromwell.backend.impl.htcondor.caching.provider.mongodb.MongoCacheActorFactory"
+    #      enabled = true
+    #      forceRewrite = false
+    #      db {
+    #        host = "127.0.0.1"
+    #        port = 27017
+    #        name = "htcondor"
+    #        collection = "cache"
+    #      }
+    #    }
+    #
+    #    filesystems {
+    #      local {
+    #        # Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
```

Comment:
I imagine this is copy / paste and wasn't updated in a while, but this is no longer true, we don't copy to `<root>/<workflow UUID>/workflow-inputs` anymore, but rather each input in its call directory.